html/obsolete/requirements-for-implementations/other-elements-attributes-and-apis/document-color-fg.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<title>document wg color</title>
+<link rel="author" title="akansha2608" href="mailto:akanshasinghal999@gmail.com">
```

Comment:
Git history provides authorship ;) this allows multiple people to work on different parts of tests over time.

Reply:
To be clear, please remove this line.

Reply:
Removed it. :dancer: